contracts/LifCrowdsale.sol

```diff
@@ -254,17 +263,51 @@ contract LifCrowdsale is Ownable, Pausable {
       foundationWallet.transfer(foundationBalanceCapWei);
 
       // TODO: create the MVM with a start block that equals one month after crowdsale ends
-      LifMarketValidationMechanism newMVM = new LifMarketValidationMechanism(
+      MVM = new LifMarketValidationMechanism(
         address(token), block.timestamp.add(10), 30 days, MVMPeriods, foundationWallet
       );
-      newMVM.fund.value(mmFundBalance)();
-      newMVM.transferOwnership(foundationWallet);
 
-      MVM = address(newMVM);
+      mintExtraTokens(MVMPeriods);
+
+      MVM.fund.value(mmFundBalance)();
+      MVM.transferOwnership(foundationWallet);
 
     }
   }
 
+  // distribute the rest of the tokens among founders and
+  function mintExtraTokens(uint8 foundationPaymentCliff) internal {
```

Comment:
The foundation 5% reserve vesting schedule doesn't have a cliff but it starts X months after it's created. X being the MVM lifetime in 30 days periods, this vesting schedule starts on `block.timestamp + X months` and has a duration of X 30 days periods.